Gopkg.toml

```diff
@@ -73,6 +73,11 @@ required = ["github.com/wadey/gocovmerge"]
   name = "google.golang.org/grpc"
   revision = "v1.3.0"
 
+[[override]]
+  name = "gopkg.in/yaml.v2"
+  source = "https://github.com/simonferquel/yaml"
```

Comment:
Perhaps add a comment, explaining why the fork is used, referring to the upstream (unmerged) PR https://github.com/go-yaml/yaml/pull/375